### PR TITLE
Gubernator fixes.

### PR DESCRIPTION
**Pull request by rmmh**
Fixes some minor problems.

### CODE REVIEW

File: gubernator/filters_test.py [hunk 1]
```diff
@@ -65,6 +65,10 @@ def test_testcmd_e2e(self):
             "go run hack/e2e.go -v -test --test_args='--ginkgo.focus="
             "Proxy\\s\\[k8s\\.io\\]\\sworks$'")
 
+    def test_testcmd_bazel(self):
+        self.assertEqual(filters.do_testcmd('//pkg/foo/bar:go_default_test'),
```
Comment (fejta) [inline gubernator/filters_test.py]: nit: line break after `assertEqual(`

File: gubernator/github/classifier.py [hunk 2]
```diff
@@ -21,6 +21,33 @@
 XREF_RE = re.compile(r'k8s-gubernator.appspot.com/build(/[^])\s]+/\d+)')
 
 
+class Deduper(object):
+    ''' Arena intern() equivalent.
```
Comment (fejta) [inline gubernator/github/classifier.py]: What is Arena intern()?

File: gubernator/github/classifier.py [hunk 3]
```diff
@@ -21,6 +21,33 @@
...
+
+    Tries to reduce memory usage by making equivalent strings point at the same object.
```
Comment (fejta) [inline gubernator/github/classifier.py]: Would this be useful for testgrid?

File: gubernator/github/classifier.py [hunk 4]
```diff
@@ -21,6 +21,33 @@
...
+    def dedup(self, obj):
+        if isinstance(obj, basestring):
+            return self.strings.setdefault(obj, obj)
+        elif isinstance(obj, dict):
+            for k in obj.keys():
+                obj[k] = self.dedup(obj[k])
+            return obj
+        elif isinstance(obj, tuple):
+            for x in obj:
+                self.dedup(x)
```
Comment (fejta) [inline gubernator/github/classifier.py]: It doesn't look like this dedups the strings in the tuple?

Maybe something like: return tuple(self.dedup(x) for x in obj)?

**Comment by spxtr** [comment]
I'm still shocked that the repeated strings are such a problem. Last two commits look great.

File: gubernator/github/handlers.py [hunk 5]
```diff
@@ -225,10 +225,16 @@ def emit_events(self, repo, number):
         return merged
 
     def get(self):
-        self.response.write(
-            '<style>td pre{max-height:200px;overflow:scroll}</style>')
         repo = self.request.get('repo')
         number = self.request.get('number')
+        if self.request.get('format') == 'json':
```
Comment (fejta) [inline gubernator/github/handlers.py]: I guess this is for debugging purposes?

**Comment by fejta** [comment]
Can you be more verbose in the PR description? Aka bazel rerun command, reduce memory footprint, add ?format=json for debugging.

/lgtm

**Comment by spxtr** [comment]
LGTM

**Pull request merged**